azure/azjwt/azjwt.go

```diff
@@ -147,12 +148,24 @@ func (a *Auth) Authenticator(next http.Handler) http.Handler {
 			return
 		}
 
+		// Add the email to the context so that it can be used by the handler
+		ctx := context.WithValue(r.Context(), emailContextKey{}, allowedEmails[0])
```

Comment:
Seems confusing to blindly choose the first one. I'd forward them all along (e.g. the value is a `[]string`), and let the downstream system decide what to do with it. I don't know what scenarios it can happen in, but I think it plays into multiple accounts (think multiple Google accounts in a single browser), so we should forward it along to the end application

Reply:
OK!